pkg/config/config.go

```diff
@@ -55,6 +55,7 @@ const DEFAULT_ROTATED_CMI_MEM_PERCENT = 63
 const DEFAULT_METADATA_MEM_PERCENT = 20
 const DEFAULT_SEG_SEARCH_MEM_PERCENT = 15 // minimum percent allocated for segsearch
 const DEFAULT_METRICS_MEM_PERCENT = 2
+const DEFAULT_MEM_PER_QUERY_BYTES = 100 * 1024 * 1024 // 100MB
```

Comment:
lets make this 200 MB based on recent tests

Reply:
addressed in the latest commit.